### ISSUE TITLE
[Bug]: How to change the number of input tokens

### ISSUE THREAD

**Issue by jxxyh** [AntCPLab/OpenBumbleBee#14]
### Issue Type

Usability

### Modules Involved

MPC protocol

### Have you reproduced the bug with SPU HEAD?

No

### Have you searched existing issues?

Yes

### SPU Version

724d3d1

### OS Platform and Distribution

Linux Ubuntu22.04

### Python Version

3.10

### Current Behavior?

I followed the method described in the README to run privacy-preserving inference with Bert and found that the shape of the input input_ids is (1,9). However, the paper states that the Bert model uses 128 tokens. Does this mean that I need to change the shape of input_ids to (1,128)?

![Image](https://github.com/user-attachments/assets/f165dab7-ac0d-4bc5-808c-5ee3444384cc)

But after padding to 128, I encountered the error shown in the image. How can I resolve this issue?

![Image](https://github.com/user-attachments/assets/6a1a385d-6607-487f-b518-399785434b49)

Additionally, if I want to measure communication cost and the rounds based on this, how should I do?

### Standalone code to reproduce the issue

```Python
repeat times = 128 // input_ids.shape[1] + 1
input_ids = jax.numpy.tile(input_ids,(1,repeat_times))[:,:128]
```

### Relevant log output

```Shell

```

**Comment by jxxyh** [comment]
I have already found a solution to the issue with the input tokens. Sorry for any confusion. 
If there is a way to measure the communication cost and rounds? Thank you!

**Comment by fionser** [comment]
* padding via the Tokenizer
* No way to measure the rounds. For the communication volumes, just see th log

**Comment by jxxyh** [comment]
Thank you
